modules/core/src/main/java/org/apache/synapse/config/xml/CallMediatorSerializer.java

```diff
@@ -46,6 +46,7 @@ public OMElement serializeSpecificMediator(Mediator m) {
         if (activeEndpoint != null) {
             call.addChild(EndpointSerializer.getElementFromEndpoint(activeEndpoint));
         }
+        call.addAttribute(fac.createOMAttribute("blocking", nullNS, Boolean.toString(mediator.getBlocking())));
```

Comment:
We don't need to add the attribute always. We can only add the attribute, if blocking is true.